### PR TITLE
Mutiple typofix and sytlefix

### PR DESCRIPTION
**Pull request by lazyparser**
Fix type fix and grammar issues in the PTS Proposal.

### CODE REVIEW

File: projects/infrastructure-for-perf-tracking.adoc [hunk 1]
```diff
@@ -13,16 +12,16 @@ Document conventions:
...
 The idea and implementation are based on the arewefastyet open-source project originated from Mozilla.
 
-After the system set up, we may need a task group for determining which hardware, which software, which benchmarks should be added into the system.
+After the system is set up, we may need a task group for determining which hardware, which software, which benchmarks should be added into the system.
 The task group may share the meeting with code speed opt SIG, for the people attending the two groups are almost the same.
 
-The main difference of the perf tracking system and general/upstream CI/CD system is that the PTS is focusing on getting real performance data, hence it should always run on physical hardware, and different hardware has different performance characteriscs.
+The main difference of the perf tracking system and general/upstream CI/CD system is that the PTS is focusing on getting real performance data, hence it should always run on physical hardware, and different hardware has different performance characteris.
```
Comment (jeremybennett) [inline projects/infrastructure-for-perf-tracking.adoc]: characteris -> characteristics

File: projects/infrastructure-for-perf-tracking.adoc [hunk 2]
```diff
@@ -34,22 +33,22 @@ Java speed on RV was ~100x slower than x86/Arm w/o JIT.
...
 So, it is better to put under the umbrella of RISC-V International.
 
 === Document history
 
 [cols="<2,<2,<3,<7",options="header,pagewidth",]
 |================================================================================
-| _Revision_ | _Date_        | _Author_ | _Modification_
-| 0.2       | 2021-1-3   |
+| _Revision_ | _Date_        | _Author_ | _Modification_
+| 0.2       | 2021-1-3   |
```
Comment (jeremybennett) [inline projects/infrastructure-for-perf-tracking.adoc]: format is 2021-01-03

File: projects/infrastructure-for-perf-tracking.adoc [hunk 3]
```diff
@@ -71,28 +70,28 @@ The performance tracking system has these type of objects:
...
+  - Add Hifive Unmatched and other physical boards, like Alibaba, etc.
+  - Add more compilers & runtimes for academic and domain specific languages, like Haskell, OCaml, Julia, R, etc.
+  - Set up APIs or processes to let RVI members run the scripts on their own devices and upload to the platform/website. (The idea is much like the submitting process of spec.org )
+  - Include OpenBLAS and other compute libs.
 
 === Outline plan and timescale
 
 * Phase 1: 1 engineer. 1 month. Plan: 2021-2-1 ~ 2021-3-1
-  - Leave one month for volunteers. Can start today, although.
+  - Leave one month for volunteers. Can start today, although.
```
Comment (jeremybennett) [inline projects/infrastructure-for-perf-tracking.adoc]: Better would be "Can start today though"

File: projects/infrastructure-for-perf-tracking.adoc [hunk 4]
```diff
@@ -128,46 +127,46 @@ It is a small project in the first place so the milestone is the phase stage its
 === Costs
 
 * equipment costs
-  - Plan to get add physical boards donated by the member companies which made them.
-  - An VPS/IaaS for hosting the website. PLCT has existing VPS and can donate for it.
+  - Plan to get more physical boards donated by the member companies which made them.
+  - An VPS/IaaS for hosting the website. PLCT has an existing VPS and can donate for it.
```
Comment (jeremybennett) [inline projects/infrastructure-for-perf-tracking.adoc]: should be "...and can donate it"

File: projects/infrastructure-for-perf-tracking.adoc [hunk 5]
```diff
@@ -128,46 +127,46 @@ It is a small project in the first place so the milestone is the phase stage its
...
 * license costs
-  - Use free & open benchmark first.
-  - The non-free benchmarks are under investigation. How to donate? Is it possible? (due the LINCENSE of each benchmarks)
+  - Use free & open benchmark first.
+  - The non-free benchmarks are under investigation. How to donate? Is it possible? (due the LICENSE of each benchmark)
 * personnel costs
-  - In the first phase a professional engineer might need 80 hours to get all things done.
-  - After the website and hardware farm established, roughly one engineer one day (8 hours) per week.
+  - In the first phase a professional engineer might need 80 hours to get all things done.
+  - After the website and hardware farm were established, roughly one engineer one day (8 hours) per week.
```
Comment (jeremybennett) [inline projects/infrastructure-for-perf-tracking.adoc]: were -> are

**Review by jeremybennett — CHANGES_REQUESTED** [review]
Since it is a typo fix commit, I've gone through and requested some small changes.

Otherwise looks good. Your English is much better than my Chinese!

File: projects/infrastructure-for-perf-tracking.adoc (hunk 1 above)
Comment (lazyparser) [reply projects/infrastructure-for-perf-tracking.adoc]: fixed

File: projects/infrastructure-for-perf-tracking.adoc (hunk 2 above)
Comment (lazyparser) [reply projects/infrastructure-for-perf-tracking.adoc]: fixed

File: projects/infrastructure-for-perf-tracking.adoc (hunk 3 above)
Comment (lazyparser) [reply projects/infrastructure-for-perf-tracking.adoc]: updated

File: projects/infrastructure-for-perf-tracking.adoc (hunk 4 above)
Comment (lazyparser) [reply projects/infrastructure-for-perf-tracking.adoc]: fixed

File: projects/infrastructure-for-perf-tracking.adoc (hunk 5 above)
Comment (lazyparser) [reply projects/infrastructure-for-perf-tracking.adoc]: fixed

**Review by jeremybennett — APPROVED** [review]
Looks good to merge

**Pull request merged**